docs/user-guide/recipes.rst

```diff
@@ -24,6 +24,14 @@ appropriate path. If you plan to serve both, Crane can scan the whole
 ``/var/lib/pulp/published/docker`` path, filtering for ``*.json`` files. Crane
 will check the ``data_dir`` for updates periodically.
 
+.. note::
+
+   As mentioned above, Crane is able to serve both content version v1 and v2, though it is
+   required to have installed proper docker client version which will be capable to fetch the content.
+   Bear in mind that in newer docker client versions interaction with V1 registries is derprecated and
```

Comment:
s/V1/v1/ for consistency. 
s/versions/versions,/
s/ and/, and/
s/derprecated/deprecated/  <--- that is hilarious though